lib/releaselib.sh

```diff
@@ -563,8 +558,16 @@ release::gcs::push_release_artifacts() {
 
   # No need to check this for mock or stage runs
   # Overwriting is ok
+  # Return without error if --allow-up flag is set
   if ((FLAGS_nomock)) && ! ((FLAGS_stage)); then
-    release::gcs::destination_empty $dest || return 1
+    if release::gcs::destination_empty $dest; then
+      if ((FLAGS_allow_dup)); then
+        logecho "flag --allow-dup set, return without error"
```

Comment:
bikeshed: s/return/exit

Reply:
it's not exiting the program though

Reply:
shouldn't it? what else are we doing after not-uploading?

Reply:
there are more stuff after https://github.com/kubernetes/release/blob/master/push-build.sh#L198

the docker images push and publish latest file pointer etc